src/plugins/intel_cpu/src/ngraph_transformations/ref_convert_i64_i32.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/pass.hpp"
+
+namespace ov {
+namespace intel_cpu {
+class RefConvertI64ToI32: public ov::pass::ModelPass {
```

Comment:
Ideally, it would be great to inherit from `MatcherPass`: I suppose this approach will be faster than the current one.
If we have some time to refactor the transformation, please let me know: I will bring you the detailed instructions offline